### PR TITLE
Scale up github receiver.

### PR DESCRIPTION
**Pull request by HumairAK**
Part of: https://github.com/operate-first/SRE/issues/108

### CODE REVIEW

**Comment by HumairAK** [comment]
/hold
We should merge this once everything is deployed again to avoid spam

**Review by tumido — APPROVED** [review]
/approve leave on hold for now, please

**Review by harshad16 — COMMENTED** [review]
/lgtm

**Pull request merged**
